pkg/operators/wasm/datasource.go

```diff
@@ -504,7 +513,7 @@ func (i *wasmOperatorInstance) dataArrayAppend(ctx context.Context, m wapi.Modul
 		stack[0] = 1
 		return
 	}
-
+	i.delHandle(dataHandle)
```

Comment:
After the append, the user won't do anything else with the data. Its to improve the UX. Create packet -> append and no manual release is needed

Reply:
Not sure and appending it to another array might also make sense.

I'll remove it since, as you said, it is not definitive that it won't be used afterwards